client/rest-high-level/src/main/java/org/opensearch/client/RequestConverters.java

```diff
@@ -266,7 +267,12 @@ static Request bulk(BulkRequest bulkRequest) throws IOException {
                     }
                 }
             } else if (opType == DocWriteRequest.OpType.UPDATE) {
-                source = XContentHelper.toXContent((UpdateRequest) action, bulkContentType, ToXContent.EMPTY_PARAMS, false).toBytesRef();
+                source = org.opensearch.core.xcontent.XContentHelper.toXContent(
```

Comment:
```suggestion
                import org.opensearch.core.xcontent.XContentHelper;
```
?

Seems like we could eliminate unnecessary formatting changes (noise on the pull request)

Reply:
Like the refactoring of the large `Strings.*`, this is incrementally refactoring `XContentHelper` so a little further below you'll see the following which is in the `org.opensearch.common.xcontent.XContentHelper` and not in the `org.opensearch.core.xcontent.XContentHelper`

```
                    XContentParser parser = XContentHelper.createParser(
                        /*
                         * EMPTY and THROW are fine here because we just call
                         * copyCurrentStructure which doesn't touch the
                         * registry or deprecation.
                         */
                        NamedXContentRegistry.EMPTY,
                        DeprecationHandler.THROW_UNSUPPORTED_OPERATION,
                        indexSource,
                        mediaType
                    )
```